libtransact/src/contract/context/mod.rs

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright 2019 Cargill Incorporated
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * -----------------------------------------------------------------------------
+ */
+
+pub mod error;
+pub mod key_value;
```

Comment:
Add #[cfg(feature = "contract-context-key-value")] and update instances of '#[cfg(feature = "contract-context")]' which are key-value specific to use the more specific feature.